lib/biscuit.rb

```diff
@@ -1,5 +1,9 @@
 require "biscuit/version"
 
 module Biscuit
-  # Your code goes here...
+  def self.run!(command)
+    result = `#{__dir__}/../bin/_biscuit #{command}`
+    raise(result) unless $?.success?
```

Comment:
I still think this is a bit dangerous - for simplicity's sake, what if you just `raise(result.slice(0,200))` instead?

The problem I've run into before is that something (pretty much always mysql2) raises an exception containing a message that is either kind of long (1k) or crazy long (20k), generally because it's including some constructed data in the message (usually a generated sql query). Then various services that store that data (a) fail to accept it because they expect exception messages to be smaller or (b) have a broken ui because even if their backend devs have learned to expect 30-page long exception messages occasionally, frontend devs never do.

We have exactly that symptom being resolved on insights right now with respect to transcription error messages from voicebase, as an [example](https://trello.com/c/l0CAtjd3/2747-bug-s4a5-transcripts-error-info-overflowed-should-probably-be-a-text-column).